### PR TITLE
First pass design

### PR DESCRIPTION
**Pull request by alexelash**
# Overview
This is a first pass at rebuilding the GeoTrellis marketing site with some minor design updates and an overhaul of the original code in order to make it easier for us (and the team) to make updates.

# Notes
- As mentioned above, there are some minor improvements (removal of transit demo from hero, more responsive site, better highlighting of current projects, usage of approved GeoTrellis branding, etc) but overall the site is close to what it is currently.
- I pulled what seemed like visually engaging screenshots for the highlighted projects (MMW, Raster Foundry, OpenTreeMap) but please let me know if there are better views! I also edited the provided text a little bit for readability / flow.
- I've added Google Tag manager to this site.

# Testing
- `git pull`
- Click through carousel, interact with inputs and ensure that they are operating as they do on geotrellis.io
- Change screen size on different page types (Homepage, Project page, Documentation page) to ensure that website looks good at various screen widths (the carousel essentially matches the current carousel's design decisions—it gives a peak of the map for visual interest but hides the interactive pane).
- Ensure that links work as expected.
- @rossbernet Would you mind reviewing the copy on all pages? I did a little reworking but definitely need an accuracy + grammar once over! Let me know if we should do this together.
- @moradology Would you mind reviewing the documentation links on the `/documentation` page?

### CODE REVIEW

File: _pages/documentation.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - text: Core Concepts
+      url: https://docs.geotrellis.io/en/latest/guide/core-concepts.html 
+
+# user guide section
+
+# scala api section
+scala-api:
+  title: Scala API Documentation
+  blurb: The latest Scaladoc tracks the master branch on github. Tagged releases are identified by their version number.
+  links:
```
Comment (moradology) [inline _pages/documentation.md]: I think we'll want to support only `0.10` through `latest` here and we probably need to think about what we want to do when the list becomes too long

File: _pages/documentation.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  body: If you can’t find exactly what you’re looking for in either our Guide or API docs, feel free to join us in our [Gitter channel,](https://gitter.im/geotrellis/geotrellis) where we can answer your questions live.
+  button:
+    text: Go to docs
+    url: https://geotrellis.readthedocs.io/en/latest/
+# intro section
+
+# user guide section
+user-guide:
+  title: GeoTrellis User Guide
+  blurb: Our [User Guide](https://geotrellis.readthedocs.io/en/latest/) is hosted on ReadTheDocs, and provides all the conceptual information required to use GeoTrellis.
```
Comment (moradology) [inline _pages/documentation.md]: We probably don't need to call out who's hosting our documentation

File: _pages/documentation.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    - text: Version 0.10
+      url: https://geotrellis.github.io/scaladocs/0.10/#geotrellis.package
+    - text: Version 0.9
+      url: https://geotrellis.github.io/scaladocs/0.9/#geotrellis.package
+    - text: Version 0.8
+      url: https://geotrellis.github.io/scaladocs/0.8/#geotrellis.package
+    - text: Version 0.7
+      url: https://geotrellis.github.io/scaladocs/0.7/#geotrellis.package
+    - text: Version 0.6
+      url: https://geotrellis.github.io/scaladocs/0.6/#geotrellis.package
```
Comment (moradology) [inline _pages/documentation.md]: Perhaps we can add a link to the github branch which corresponds to the API docs

File: _sass/08_overrides/_projects.scss [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    .project-text {
+        @media all and (max-width: $bp-large-2) {
+            padding: $pad-comfortable;
+        }
+    }
+
+    .page-header {
+
+        &__title {
+            margin-bottom: 0;
```
Comment (designmatty) [inline _sass/08_overrides/_projects.scss]: This makes the text feel really tight. I might remove this margin to let that section breath a little bit.

File: _includes/_home/_intro.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<section class="section section--intro">
+    <div class="section__inner">
+        <h1 class="section__callout" markdown="1">
```
Comment (designmatty) [inline _includes/_home/_intro.html]: Woah, I didn't know about the `markdown="1"` feature. Very cool

**Comment by moradology** [comment]
We should ensure that altitude changes are propagated to the position of the sun in our hillshade demo

File: _projects/model-my-watershed.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: project
+
+# NO EDITING ABOVE THIS LINE
+# # # # # #
+
+seo_description: "GeoTrellis runs the hydrologic models and returns results to students as they make simulated changes to the landscape in the Model My Watershed application, created with the Stroud Water Resource Center."
```
Comment (moradology) [inline _projects/model-my-watershed.md]: : "GeoTrellis runs hydrologic models and returns results to students as they mak changes to the simulated landscape in the Model My Watershed application, created with the Stroud Water Resource Center."

File: _projects/model-my-watershed.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+title: Model My Watershed
+featured-image: ./assets/images/feature_model-my-watershed.jpg
+featured-image-alt: Screenshot of an outlined watershed in Model My Watershed.
+page_class: l-project
+order: 1
+
+# # # # # #
+# Page copy
+# # # # # #
+intro: The Stroud Water Resource Center received funding from the National Science Foundation to create new educational tools that will help students learn by interacting with watershed processes in a game-like environment.
```
Comment (moradology) [inline _projects/model-my-watershed.md]: "The Stroud Water Resource Center received funding from the National Science Foundation to create new educational tools that will help students learn about watershed processes in a game-like environment."

File: _projects/model-my-watershed.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+body: GeoTrellis runs hydrologic models and quickly returns results as student users adjust parameters in Model My Watershed. This allows them to simulate changes to the landscape and then visualize the impact of these changes on the local ecosystem. Students are also able to draw custom bounding boxes and query against several raster datasets at web speed. Read more about the project [here.](https://wikiwatershed.org/documentation/mmw-tech/)
```
Comment (moradology) [inline _projects/model-my-watershed.md]: ' at web speed' I think we can strike this (seems like a show, don't tell sort of thing)

File: _projects/opentreemap.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+title: "OpenTreeMap"
+featured-image: ./assets/images/feature_otm.jpg
+featured-image-alt: "Screenshot of trees on a map in OpenTreeMap."
+page_class: l-project
+order: 3
+
+# # # # # #
+# Page copy
+# # # # # #
+intro: OpenTreeMap allows urban foresters to map trees from a phone or tablet, calculate ecosystem benefits and more.
```
Comment (moradology) [inline _projects/opentreemap.md]: Oxford comma, imo

**Comment by moradology** [comment]
I've got a few comments worth looking at. Beyond those, everything looks really, really good. +1 from our side

**Comment by designmatty** [comment]
Make sure to remove the `static` folder since our static is now compiled into `_site`

File: _includes/_home/_intro.html (hunk 5 above)
Comment (alexelash) [reply _includes/_home/_intro.html]: It's super useful!

File: _projects/model-my-watershed.md (hunk 6 above)
Comment (alexelash) [reply _projects/model-my-watershed.md]: Also fixed my run-on sentence.

File: _sass/08_overrides/_projects.scss (hunk 4 above)
Comment (alexelash) [reply _sass/08_overrides/_projects.scss]: You right. Resolved in c7de34f.

**Comment by alexelash** [comment]
Cool, thanks everyone. This is ready for another look.

@moradology Could you review the documentation page one last time, given the new changes?
@designmatty Ready for another design once-over.

**Pull request merged**